recipes/tachyon/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+cd unix
+case "$(uname)" in
+    Darwin) target=macosx-x86_64-thr;;
+    Linux) target=linux-64-thr;;
+esac
+
+make $target -j$CPU_COUNT || make $target -j$CPU_COUNT
```

Comment:
Sorry, just ignored that one. Seen enough build systems that are flaky the first time around, especially when run with `-j` as that's non-deterministic.